src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -13,6 +13,9 @@
     <None Include="$(MSBuildThisFileDirectory)CSharpAnalyzersResources.resx" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(MSBuildThisFileDirectory)ConvertSwitchStatementToExpression\ConvertSwitchStatementToExpressionConstants.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ConvertSwitchStatementToExpression\ConvertSwitchStatementToExpressionDiagnosticAnalyzer.Analyzer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)ConvertSwitchStatementToExpression\ConvertSwitchStatementToExpressionDiagnosticAnalyzer.cs" />
```

Comment:
@sharwell @mavasani Do we need to do this for any new analyzer? Hopefully not..

Reply:
@tmat is taking care of this in https://github.com/dotnet/roslyn/pull/42517. However, please add new analyzers to this shared project, so they automatically get included in the CodeStyle NuGet package and are available in CI.

Reply:
also tagging @CyrusNajmabadi